package/src/index.ts

```diff
@@ -70,6 +114,260 @@ export class CandleClient {
   }): Promise<string> {
     return this.candle.executeTool(tool);
   }
+
+  public async getAssetAccounts(
+    query: AssetAccountQuery = {
+      linkedAccountIDs: undefined,
+      assetKind: undefined,
+    }
+  ): Promise<AssetAccount[]> {
+    const accounts = await this.candle.getAssetAccounts(query);
+    return accounts.map(this.convertToAssetAccount);
+  }
+
+  public async getTrades(
+    query: {
+      gainedAssetKind?: TradeQueryAssetKind;
+      lostAssetKind?: TradeQueryAssetKind;
+      counterpartyKind?: "merchant" | "user" | "service";
+    } & TradeQuery = {
+      counterpartyKind: undefined,
+      gainedAssetKind: undefined,
+      lostAssetKind: undefined,
+      linkedAccountIDs: undefined,
+      dateTimeSpan: undefined,
+    }
```

Comment:
#nth same as above, this is unnecessary and would be a linter error
```suggestion
    } & TradeQuery = {}
```